docs/docs/new-features/index.mdx

```diff
@@ -1,7 +1,272 @@
 ---
 sidebar_position: 4
+hide_table_of_contents: true
 ---
 
+import Link from "@docusaurus/Link";
+import { CardGroup, NewFeatureCard } from "@site/src/components/Card";
+import { APILink } from "@site/src/components/APILink";
+
 # New Features
 
-{/* WIP */}
+<CardGroup>
+  <NewFeatureCard
+    description={
+      <>
+        <Link to="/llms/langchain">LangGraph</Link>, the GenAI Agent authoring framework from LangChain, is now natively supported in MLflow using the <Link to="/model/models-from-code">Models from Code</Link> feature.
+      </>
+    }
+    name="LangGraph Support"
+    releaseVersion="2.16.0"
+  >
+    <span>![LangGraph](/images/logos/langgraph-logo.png)</span>
```

Comment:
could we add a `max-width: 80%` to all these spans? the images look a bit too wide to me